tests/phpunit/tests/functions/referer.php

```diff
@@ -18,8 +18,6 @@ public function setUp() {
 	}
 
 	public function tearDown() {
-		$_SERVER['HTTP_REFERER']      = '';
-		$_SERVER['REQUEST_URI']       = '';
```

Comment:
Similar remark as before:
`$_SERVER` **is** being (conditionally) reset via `tests_reset__SERVER()` in `WP_UnitTestCase_Base::reset__SERVER()` which is called in `WP_UnitTestCase_Base::setUp()`, but that is _unstable_.